backend/src/main/java/io/github/k_gregory/registry/controller/IndividualController.java

```diff
@@ -0,0 +1,123 @@
+package io.github.k_gregory.registry.controller;
+
+import io.github.k_gregory.registry.infrastructure.ResourceNotFoundException;
+import io.github.k_gregory.registry.model.Individual;
+import io.github.k_gregory.registry.repository.IndividualRepository;
+import org.modelmapper.ModelMapper;
+import org.modelmapper.TypeToken;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Pattern;
+import javax.validation.constraints.Size;
+import java.lang.reflect.Type;
+import java.sql.Date;
+import java.util.List;
+
+class IndividualDto {
+    @NotNull
+    @Size(min = 10, max = 10)
+    private String uid;
+
+    @NotNull
+    private String firstName;
+
+    @NotNull
+    private String middleName;
+
+    @NotNull
+    private String lastName;
+
+    @NotNull
+    private Date birthDate;
+
+    @NotNull
+    @Pattern(regexp = "^(?:(?![ЫЪЭ])[А-ЯЄІЇ]){2}\\d{6}$", flags = Pattern.Flag.UNICODE_CASE)
```

Comment:
@k-gregory Could you please setup the tests infrastructure since I am not so keen in java unit tests libraries and their integration with gradle. Will add the tests after the infrastructure creation. Could do that in separate PR

Reply:
Okay, I feel obligated to implement this infrastructure as soon as possible)